roswallp.txt

```diff
@@ -0,0 +1,15 @@
+[Section]
+Name = ReactOS Standard Wallpapers
+Version = 2
+License = CC BY
+Description = The standard wallpaper collection for ReactOS
+Category = 16
+URLSite = https://github.com/reactos/wallpapers/
+URLDownload = https://github.com/reactos/wallpapers/archive/refs/heads/main.zip
+Installer = Generate
+SizeBytes = 7479362
+
+[Generate]
+Files = wallpapers-main\data\*
+Dir = %SystemRoot%\Web\Wallpaper
+Lnk = !
```

Comment:
Is it part of the new syntax from @whindsaks 's PR?
If so, I have a question for @whindsaks : Could you remind me quickly what this syntax does? And what would happen if this `Lnk = !` would be removed here?

Reply:
It means don't create a start menu shortcut. If you remove it, it will create a shortcut in the start menu to the first item extracted.